src/prefect/server/schemas/actions.py

```diff
@@ -860,6 +860,9 @@ class WorkQueueUpdate(ActionBaseModel):
     concurrency_limit: Optional[NonNegativeInteger] = Field(None)
     priority: Optional[PositiveInteger] = Field(None)
     last_polled: Optional[DateTimeTZ] = Field(None)
+    status: Optional[schemas.statuses.WorkQueueStatus] = Field(
+        default=None, description="The current status of the work queue."
+    )
```

Comment:
Yeah I thought that was odd too, it's in Prefect Cloud that way too, but it seems wrong to me as well.

https://github.com/PrefectHQ/nebula/blob/main/src/prefect/server/schemas/actions.py#L1013-L1015

I'm inclined to drop it from both sides since it's not in OSS (and thus we aren't likely using it)